jfxtras-controls/src/main/java/jfxtras/scene/control/LocalDateTimeTextField.java

```diff
@@ -187,6 +187,36 @@ public void set(DateTimeFormatter value) {
 	public void setText(String value) { textObjectProperty.set(value); }
 	public LocalDateTimeTextField withText(String value) { setText(value); return this; }
 	
+        

```

Comment:
I my opinion the setter, getter and wither are trivial and do not require documentation if the property is documented suffiently. It, after all, has a setter and getter.